src/cascadia/UnitTests_Control/ControlCoreTests.cpp

```diff
@@ -499,4 +501,68 @@ namespace ControlUnitTests
             VERIFY_ARE_EQUAL(expectedEnd, end);
         }
     }
+
+    void ControlCoreTests::TestSimpleClickSelection()
+    {
+        // Create a simple selection with the mouse, then click somewhere else,
+        // and confirm the selection got updated.
+
+        auto [settings, conn] = _createSettingsAndConnection();
+        Log::Comment(L"Create ControlCore object");
+        auto core = createCore(*settings, *conn);
+        VERIFY_IS_NOT_NULL(core);
+        _standardInit(core);
+
+        // Here, we're using the UpdateSelectionMarkers as a stand-in to check
+        // if the selection got updated with the renderer. Standing up a whole
+        // dummy renderer for this test would be not very ergonomic. Instead, we
+        // are relying on ControlCore::_updateSelectionUI both
+        // TriggerSelection'ing and also rasing this event
+        bool expectedSelectionUpdate = false;
+        bool gotSelectionUpdate = false;
+        core->UpdateSelectionMarkers([&](auto&& /*sender*/, auto&& /*args*/) {
+            VERIFY_IS_TRUE(expectedSelectionUpdate);
+            expectedSelectionUpdate = false;
+            gotSelectionUpdate = true;
+        });
+
+        auto needToCopy = false;
+        expectedSelectionUpdate = true;
+        core->LeftClickOnTerminal(til::point{ 1, 1 },
+                                  1,
+                                  false,
+                                  true,
+                                  false,
+                                  needToCopy);
+
+        VERIFY_IS_TRUE(core->HasSelection());

```

Comment:
wait, why now does a single left click result in a selection? I thought it for sure didn't do that...

Reply:
that `true` means shift was pressed. A single click with a shift _does_ select